### PR TITLE
Enable SPROT server on the xpressoboard

### PR DESCRIPTION
**Pull request by labbott**
The xpressoboard doesn't actually have the hookups to be able to connect to the SP but it's more likely to do something than the old SPI driver

### CODE REVIEW

**Review by lzrd — COMMENTED** [review]
Is this tested? (I don't see any reason it would not work)

**Comment by lzrd** [comment]
You tracked down the P20 (PMOD) connector and P1 jumper on the xpresso board which would allow use without soldering.

Laura writes:
"...access for a remote host to the SPI and I2C ports of the LPC55Sxx that support ISP mode....it uses SPI3..." "When using the SPI port at this connector ensure jumper P1 is installed to disable the connection to the Link2 debug probe. ... installed on my board by default as well..."

File: app/lpc55xpresso/app.toml [hunk 1]
```diff
@@ -151,3 +128,30 @@ start = true
...
+# HS_SPI_MISO = P1_3 = FUN6
+{ pin = { port = 1, pin = 3}, alt = 6},
+# HS_SPI_SCK = P1_2 = FUN6
+{ pin = { port = 1, pin = 2}, alt = 6},
+# HS_SSEL1 = P1_1 = FUN5
+{ pin = { port = 1, pin = 1}, alt = 5},
+# ROT_IRQ = P0_18 = FUN0
+{ name = "ROT_IRQ", pin = { port = 0, pin = 18}, alt = 0, direction = "output"},
+# SP_RESET = P0_9 = FUN0
+{ name = "SP_RESET", pin = { port = 0, pin = 9}, alt = 0, direction = "input"},
```
[the remark this answers is not in the document]
Comment (lzrd) [reply app/lpc55xpresso/app.toml:155]: Pin 0_9 is SD0_PWR_EN, an output from the LPC55S69. Another reason to remove it.
Pin 0_18 shows up on xpresso board connector P18 pin 20 and can remain in this toml file.

[the remark this answers is not in the document]
Comment (labbott) [reply app/lpc55xpresso/app.toml]: I double checked and I don't see a connection to the reset pin on the gimletlet board (our expected target for sprot on this setup). `ROT_IRQ` is just a GPIO so we can always change that to another value if needed.

**Review by lzrd — APPROVED** [review]

**Pull request merged**